src/WebAppDIRAC/WebApp/handler/MonitoringHandler.py

```diff
@@ -1,257 +0,0 @@
-import os
```

Comment:
This should stay!

Reply:
It does need modifying though to remove things like: `MonitoringClient().getActivitiesContents`

Reply:
@chrisburr I don't find that in MonitoringHandler... I think it's activityMonitoringHandler